src/mscorlib/shared/System/IO/PathInternal.Unix.cs

```diff
@@ -43,6 +43,22 @@ internal static bool IsDirectorySeparator(char c)
         }
 
         /// <summary>
+        /// Returns true if the path is too long
+        /// </summary>
```

Comment:
I do not think we need any of these changes. They are unused code that was deleted in the CoreCLR version. The cleanup should be propagated to the CoreRT copy instead.